src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -788,7 +795,106 @@ private String getRouterId() {
      * startup.
      */
     private String constructUserData(boolean graphAlreadyBuilt) {
```

Comment:
Suggestion: rename to something like `makeOtpRunnerScript(...)`

Reply:
I think this can stay how it is, but perhaps the javadoc could make a mention of otp-runner.

Reply:
Yeah, I still think it makes sense to call it `constructUserData` because there are still a few bash items that are included in addition to the otp-runner script execution.

Reply:
What was the `userData` this is referring to?

Reply:
Again that was a suggestion so I won't hold the PR hostage to that.